skyplane/cli/cli.py

```diff
@@ -169,6 +173,28 @@ def cp(
         else:
             typer.secho("Transfer not supported", fg="red")
             return 1
+
+    elif provider_src == "pre-signed-url" or provider_dst == "pre-signed-url":
+        try:
+            if provider_src == "pre-signed-url":
+                dst_client = ObjectStoreInterface.create(dst_region_tag, bucket_dst)
+                dst_region_tag = dst_client.region_tag()
+                if cloud_config.get_flag("requester_pays"):
+                    dst_client.set_requester_bool(True)
+            elif provider_dst == "pre-signed-url":
```

Comment:
Will `provider_src` and `provider_dst` ever both be `pre-signed-url`? If so, the `src_client` and `src_region_tag` will be unset.

Reply:
pre-signed-url will be a different PR when implemented.